Cargo.toml

```diff
@@ -4,13 +4,13 @@ description = "Parses hostlist expressions into a Vec of Strings"
 repository = "https://github.com/whamcloud/hostlist-parser"
 keywords = ["hostlist", "pdsh", "hpc", "cluster"]
 license = "MIT"
-version = "0.1.3"
-authors = ["IML Team <iml@whamcloud.com>"]
-edition = "2018"
+version = "0.1.4"
```

Comment:
Version should change to 0.2 because this breaks the old API.

Reply:
No, anything in semver under 1.0 is a breaking change, it doesn't matter if it's major, minor or patch.